screens/MeetYourMatch.js

```diff
@@ -10,6 +11,14 @@ export class MeetMatch extends React.Component {
   state = {
     fontLoaded: false,
   };
+
+  componentWillMount() {
+    const gameId = this.props.navigation.state.params.id
+    getGame(gameId).then((game) => {
+      this.setState({game})
+      this.setState({name: this.state.game.name, place: this.state.})
```

Comment:
You don't have any `state` variable called `name` or `place`. Either remove this line or create and properly update these variables